internal/adapters/runtime/kubernetes/scheduler.go

```diff
@@ -209,14 +209,27 @@ func (k *kubernetes) MitigateDisruption(
 		}
 	}
 
-	currentPdbMinAvailable := pdb.Spec.MinAvailable.IntVal
+	var currentPdbMinAvailable int32
+	// PDB might exist and is based on MaxUnavailable
+	if pdb.Spec.MinAvailable != nil {
+		currentPdbMinAvailable = pdb.Spec.MinAvailable.IntVal
+	}
+
 	if currentPdbMinAvailable == int32(float64(roomAmount)*incSafetyPercentage) {
 		return nil
 	}
 
-	pdb.Spec.MinAvailable = &intstr.IntOrString{
-		Type:   intstr.Int,
-		IntVal: int32(float64(roomAmount) * incSafetyPercentage),
+	pdb.Spec = v1Policy.PodDisruptionBudgetSpec{
+		MaxUnavailable: nil,
```

Comment:
is this required, even if max unavailable by default is nil?

Reply:
No, I can delete since I'm creting a new struct. Will remove